<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="b4ab715b-0908-4735-b4f5-25108748d148">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/mysite/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\himna\PycharmProjects\teamAREver2\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="false" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="Django testdb" uuid="e9382de5-2884-4440-ac31-9cad63326ffc">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/mysite/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\himna\PycharmProjects\teamAREver2\dbms.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="false" />
      </driver-properties>
    </data-source>
  </component>
</project>